src/javafundamentals/collections/list/Ex04DeleteElements.java

```diff
@@ -1,9 +1,24 @@
 package javafundamentals.collections.list;
 
+
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * Create an List with numbers from 1 to 10
  * Delete the number 7 and print the list
  */
 
 public class Ex04DeleteElements {
+    public static void main(String[] args) {
+        List<Integer> numberList = new ArrayList<>();
+
+        for (int i = 1; i <= 10; i++) {
+            numberList.add(i);
+        }
+
+        numberList.remove(numberList.indexOf(7));
```

Comment:
there is another method that removes the Object that is passed on.
So if you create an Integer object you can pass it to remove the object without having to use the `indexOf()` method
`            numberList.remove(new Integer(7));
`

Reply:
Right I see, thanks!